Mathlib/Data/Finset/Pointwise.lean

```diff
@@ -2038,7 +2038,8 @@ theorem toFinset_one : (1 : Set α).toFinset = 1 :=
 #align set.to_finset_one Set.toFinset_one
 #align set.to_finset_zero Set.toFinset_zero
 
-@[to_additive (attr := simp)]
+-- Porting note: should take priority over `Finite.toFinset_singleton`
+@[to_additive (attr := simp high)]
```

Comment:
As an explanation of what's going on here; lean reduces the definition of `(1 : Set _)` because it's hidden in an implicit argument. As a mwe:
```lean
import Mathlib.Data.Set.Pointwise.Finite
import Mathlib.Data.Finset.Pointwise


namespace Set

open scoped Pointwise

variable {α : Type _} [One α]

/-- A version of `toFinset` with an explicit `s` argument. -/
noncomputable def Finite.toFinset' (s : Set α) (hs : s.Finite) : Finset α := hs.toFinset

/-- A copy of `Finite.toFinset_singleton` -/
@[simp] lemma Finite.toFinset'_singleton (a : α) (hs : Set.Finite {a} := finite_singleton _) : hs.toFinset' _ = {a} :=
  Finite.toFinset_singleton _

-- ok
@[simp] theorem Finite.toFinset'_one (h : (1 : Set α).Finite := finite_one) : h.toFinset' _ = 1 :=
  Finite.toFinset_singleton _

-- rejected by linter
@[simp] theorem Finite.toFinset_one (h : (1 : Set α).Finite := finite_one) : h.toFinset = 1 :=
  Finite.toFinset_singleton _

end Set

#lint
```

I think this probably happened in Lean 3 too, but there the simp priority was automatically correct.